README.md

```diff
@@ -169,13 +169,15 @@ When composed with `vmap`, `grad` can be used to compute per-sample-gradients:
 
 ### vjp and jacrev
 
+The `vjp` transform applies `func` to `inputs` and returns a new function that
+computes vjps given some `cotangents` Tensors.
 ```
 >>> from functorch import vjp
 >>> outputs, vjp_fn = vjp(func, inputs); vjps = vjp_fn(*cotangents)
 ```
-The `vjp` transform applies `func` to `inputs` and returns a new function that
-computes vjps given some `cotangents` Tensors.
 
+The `jacrev` transform returns a new function that takes in `x` and returns the
```

Comment:
this just fixes it so all descriptions of what's about to happen come before the code example